### DISCUSSION TITLE
Batterie nachts sogar teilweise geladen nutzen - Ich brauche eine Beschreibung

### DISCUSSION THREAD

**Discussion by Manos1966** [hoylabs/OpenDTU-OnBattery#822] [Q&A] [answered]
Kann jemand mir erklären was genau "Batterie nachts sogar teilweise geladen nutzen" macht?

Ich gehe davon aus, es deaktiviert Nachts:
- Batterienutzung Start-Schwellwert %
- Batterienutzung Start-Schwellwert V

Korrekt? Gibt es noch etwas?

EDIT: The switch between daytime and nighttime operation is based on the Victron MPPT charger controller(s) power. An output of 20W or more is interpreted as daytime.

**Comment by schlimmchen** [comment, marked as answer]
Wenn diese Option aktiviert ist, wird ein Entladezyklus gestartet, wenn es Nacht wird, obwohl die Batterie-Start-Schwelle nicht erreicht ist. Der Entladezyklus wird beendet, wenn die Nacht vorrüber ist.

Und "Nacht" ist wie du zitiert hast definiert.

Im Grunde hast du also Recht. Allerdings finde ich die Erklärung nicht ideal, auch weil ich es wichtig finde vom Lade- und Entladezyklus zu sprechen, der im Wiki definiert ist: https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Dynamic-Power-Limiter

**Reply by Manos1966** [reply]
Danke! 👍

**Comment by huschke3** [comment]
Kurze Nachfrage Zu: "Der Entladezyklus wird beendet, wenn die Nacht vorüber ist...."

Ich habe gestern den Schalter erstmalig angeschaltet und es hat mir heute Nacht den Speicher komplett entladen (BMS abgeschaltet). 
D. h. die Stop-Schwelle ist dann auch nicht mehr aktiv?

**Reply by Manos1966** [reply]
Doch! die Stop-Schwelle bleibt aktiv.

"BMS abgeschaltet" also, keine Werte ueber CANbus??? Dann hoffe ich, dass du richtige Stop-Spannungswerte definiert hast.

**Reply by huschke3** [reply]
Danke Manos1966. Dann werde ich weiter beobachten. Ich messe Spannung am Eingang 1 des WR. Lief bisher perfekt. Nun Update von 06.2024 auf aktuelle Version und den Schalter " nachts sogar teilweise an" und schwupps leer. Schwelle liegt gut 2 Volt über BMS Abschaltung.

**Reply by schlimmchen** [reply]
> es hat mir heute Nacht den Speicher komplett entladen (BMS abgeschaltet).

Das ist lästig, keine Frage, aber an diesem Schaler (alleine) wird es nicht gelegen haben.

Bitte teile deine DPL Einstellungen (Screenshots) und sag uns, welche Batterie du verwendest. Außerdem ausführliche Protokollierung einschalten und Logs zeigen, wenn der Inverter läuft obwohl er nicht sollte.
